src/Sulu/Bundle/MediaBundle/Tests/Functional/Controller/CollectionControllerTest.php

```diff
@@ -586,10 +629,14 @@ public function testPut()
 
         $this->assertNotEmpty($response);
 
-        $this->assertEquals(1, $response->total);
+        $this->assertEquals(2, $response->total); // 1 created 1 system collection
```

Comment:
I think we have to find a way around this... If another bundle adds more system collections, we also have to update these tests...